binding/HarfBuzzSharp.Shared/Definitions.cs

```diff
@@ -23,71 +16,84 @@ public enum MemoryMode
 		ReadOnlyMayMakeWriteable
 	}
 
-	[StructLayout(LayoutKind.Sequential)]
+	[StructLayout (LayoutKind.Sequential)]
 	public struct Feature
 	{
-		private hb_tag_t tag;
+		private Tag tag;
 		private uint val;
 		private uint start;
 		private uint end;
 
-		public hb_tag_t Tag
+		public Feature (Tag tag, uint value, uint start = 0, uint end = uint.MaxValue)
 		{
+			this.tag = tag;
+			this.val = value;
+			this.start = start;
+			this.end = end;
+		}
+
+		public Tag Tag {
 			get { return tag; }
 			set { tag = value; }
 		}
 
-		public uint Value
-		{
+		public uint Value {
 			get { return val; }
 			set { val = value; }
 		}
 
-		public uint Start
-		{
+		public uint Start {
 			get { return start; }
 			set { start = value; }
 		}
 
-		public uint End
-		{
+		public uint End {
 			get { return end; }
 			set { end = value; }
 		}
 
+		public override string ToString ()
+		{
+			const int allocatedSize = 128;
+			var buffer = new StringBuilder (allocatedSize);
+			HarfBuzzApi.hb_feature_to_string (ref this, buffer, allocatedSize);
+			return buffer.ToString ();
+		}
 	}
 
-	[StructLayout(LayoutKind.Sequential)]
+	[StructLayout (LayoutKind.Sequential)]
 	public struct GlyphInfo
 	{
 		private hb_codepoint_t codepoint;
 		private hb_mask_t mask;
-		private uint cluster;
+		private int cluster;
```

Comment:
I just realized that uint isn't very common in the. Net world. You always have to cast it to int to make this usable.

Also this: https://docs.microsoft.com/de-de/dotnet/api/system.char.convertfromutf32?view=netframework-4.7.2

The .Net Framework itself uses int for that data.

I just thought this would be more convenient. This change isn't easy. Maybe I should just map everything 1:1 and let the consumer of the API cast everything. Just let me know if I should change everything back to uint.